packages/fuels/tests/types.rs

```diff
@@ -1677,3 +1678,103 @@ async fn test_b512() -> Result<()> {
 
     Ok(())
 }
+
+#[tokio::test]
+async fn test_base_type_in_vec_output() -> Result<()> {
+    setup_contract_test!(
+        Wallets("wallet"),
+        Abigen(
+            name = "VectorOutputContract",
+            abi = "packages/fuels/tests/types/vector_output"
+        ),
+        Deploy(
+            name = "contract_instance",
+            contract = "VectorOutputContract",
+            wallet = "wallet"
+        ),
+    );
+    let contract_methods = contract_instance.methods();
+    let response = contract_methods.u8_in_vec(10).call().await?;
+    assert_eq!(response.value, (0..10).collect::<Vec<_>>());
+    let response = contract_methods.u16_in_vec(11).call().await?;
+    assert_eq!(response.value, (0..11).collect::<Vec<_>>());
+    let response = contract_methods.u32_in_vec(12).call().await?;
+    assert_eq!(response.value, (0..12).collect::<Vec<_>>());
+    let response = contract_methods.u64_in_vec(13).call().await?;
+    assert_eq!(response.value, (0..13).collect::<Vec<_>>());
+    let response = contract_methods.bool_in_vec().call().await?;
+    assert_eq!(response.value, [true, false, true, false].to_vec());
+    Ok(())
+}
+#[tokio::test]
+async fn test_composite_types_in_vec_output() -> Result<()> {
+    setup_contract_test!(
+        Wallets("wallet"),
+        Abigen(
+            name = "VectorOutputContract",
+            abi = "packages/fuels/tests/types/vector_output"
+        ),
+        Deploy(
+            name = "contract_instance",
+            contract = "VectorOutputContract",
+            wallet = "wallet"
+        ),
+    );
+    let contract_methods = contract_instance.methods();
+
+    {
+        let expected: Vec<[u64; 4]> = vec![
+            ([1, 1, 1, 1]),
+            ([2, 2, 2, 2]),
+            ([3, 3, 3, 3]),
+            ([4, 4, 4, 4]),
+        ];
```

Comment:
Parentheses are not needed.
```suggestion
        let expected: Vec<[u64; 4]> = vec![[1, 1, 1, 1], [2, 2, 2, 2], [3, 3, 3, 3], [4, 4, 4, 4]];
```